cmd/podman/common/completion.go

```diff
@@ -1183,6 +1183,38 @@ func AutocompleteFormat(o interface{}) func(cmd *cobra.Command, args []string, t
 			return nil, cobra.ShellCompDirectiveNoFileComp
 		}
 
+		// add suggestions for: podman inspect (it will default to container inspect)
+		switch reflect.TypeOf(o).String() {
+		case "*define.InspectContainerData":
+			if containers, _ := getContainers(cmd, args[0], completeDefault); len(containers) == 0 { // there might be a panic here when args length is 0. Is it assured that args will not be empty?
```

Comment:
you have to make a len(args) > 0 check before, if args == 0 we can assume the container type.